content/chat/rooms/messages.textile

```diff
@@ -69,10 +77,10 @@ The following are the properties of a message:
 
 h3(#unsubscribe). Unsubscribe from messages
 
-blang[javascript].
+blang[javascript,swift].
   Unsubscribe from messages to remove previously registered listeners.
 
-  Use the "@unsubscribe()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat-js.MessageSubscriptionResponse.html#unsubscribe function returned in the @subscribe()@ response to remove a listener:
+  Use the <span lang="javascript">"@unsubscribe()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat-js.MessageSubscriptionResponse.html#unsubscribe</span><span lang="swift">"@unsubscribe()@":https://sdk.ably.com/builds/ably/ably-chat-swift/main/typedoc/interfaces/chat.MessageSubscription.html#unsubscribe</span> function returned in the @subscribe()@ response to remove a listener:
```

Comment:
My Swift isn't great, but I don't think the subscription returns an unsubscribe function like in JS, and it's an independent method.

Reply:
As I understood we decided to go with the same descriptions for listeners. Or was it only for creation of the subscription/listener?

Reply:
@m-hulbert

Reply:
fc78ae83 WDYT @umair-ably @m-hulbert @lawrence-forooghian @ttypic @AndyTWF

Reply:
It makes sense for the creation of the subscription for sure. But correct me if I'm wrong, but this doesn't make sense for Swift.

Reply:
@m-hulbert done https://github.com/ably/docs/pull/2315/commits/fa775da124d425939fd853a2f8e07e571b1b5571